### DISCUSSION TITLE
GitHub 的卡片堆叠样式

### DISCUSSION THREAD

**Discussion by afc163** [ant-design/ant-design#42171] [Ideas]
<img width="739" alt="图片" src="https://user-images.githubusercontent.com/507615/236595084-edda408d-9bb7-4d37-a9de-1b1d1184fbef.png">

简单设计下 API。

```
<Card.Group>
  <Card />
  <Card />
  <Card />
</Card.Group>
```

**Comment by afc163** [comment]
下周出个 RFC

**Comment by huangkairan** [comment]
堆叠的话，是不是要有切换卡片的功能和堆叠数量？

**Reply by afc163** [reply]
要有展开收起功能。切换不一定要有。

**Comment by afc163** [comment]
https://dribbble.com/shots/19646724-Notes-Card-Stack-Candidate-Profile-Notes-Dark-Light-Mode
